src/pages/rt-fees-schedule/window/FeesSceduleWindow.js

```diff
@@ -0,0 +1,46 @@
+// ** Custom Imports
+import CustomTabPanel from 'src/components/Shared/CustomTabPanel'
+import { CustomTabs } from 'src/components/Shared/CustomTabs'
+import { useState } from 'react'
+import FeesSceduleForm from '../forms/FeesSceduleForm'
+import FeesDetailsForm from '../forms/FeesDetailsForm'
+
+const FeesSceduleWindow = ({ height, recordId, labels, maxAccess, expanded }) => {
```

Comment:
is height necessary?? please remove height from the whole file it seems that serving no purpose